infrastructure/sentry-subgraph/src/pool-factory.ts

```diff
@@ -188,14 +190,14 @@ export function handleUnstakeKeys(event: UnstakeKeys): void {
     }
 
     let index = decoded.toTuple()[1].toBigInt()
-    let unstakeRequest = UnstakeRequest.load(event.params.pool.toHexString() + event.params.user.toHexString() + index.toString())
+    let unstakeRequest = UnstakeRequest.load(poolAddress + userAddress + index.toString())
     if (unstakeRequest) {
       unstakeRequest.open = false
       unstakeRequest.completeTime = event.block.timestamp
       unstakeRequest.save();
     } else {
       log.warning("handleUnstakeKeys - Could not find unstake key request!", [])
-      log.warning("pool: " + event.params.pool.toHexString() + ", user: " + event.params.user.toHexString() + ", index: " + index.toString() + ", TX: " + event.transaction.hash.toHexString(), [])
```

Comment:
Did we ever error here ?
I could not find `handleUnstakeKeys - Could not find unstake key request!` in the graph errors logs.

I don't think the to lowercase actually makes a difference on the graph since `toHexString()` will not ever output upper cased hex chars.

Reply:
See other comment, once confirmed it's good, I will note and update PR.